global-test.properties

```diff
@@ -18,6 +18,12 @@ intersmash.wildfly.operators.index_image=quay.io/operatorhubio/catalog:latest
 intersmash.wildfly.operators.package_manifest=wildfly
 intersmash.wildfly.operators.channel=alpha
 
+# ActiveMQ community operator settings
+intersmash.activemq.operators.catalog_source=intersmash-activemq-operator-index
+intersmash.activemq.operators.index_image=quay.io/jbossqe-eap/intersmash-activemq-operator-catalog:v1.0.11
```

Comment:
Just a question here, is the version you used (v1.0.11) derived from some Artemis operator tag or did you reate it randomly (I assume not)?

Reply:
it was the default version used when building the upstream repo

Reply:
I see, thanks. Which branch have we built?

Reply:
main, now I've rebuilt everything using the latest tag